cglib/src/main/java/net/sf/cglib/core/KeyFactory.java

```diff
@@ -191,13 +207,23 @@ public void generateClass(ClassVisitor v) {
             e.super_invoke_constructor();
             e.load_this();
             for (int i = 0; i < parameterTypes.length; i++) {
-                seed += parameterTypes[i].hashCode();
+                Type parameterType = parameterTypes[i];
```

Comment:
Do you mean to move "field emission" to customizer? I just tried to keep the changes to the minimal sufficient set.